proxies/live/apiproxy/policies/Quota.Binary.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Quota name="Quota.Binary" type="rollingwindow">
+    <DisplayName>Quota.Binary</DisplayName>
+    <Identifier ref="developer.app.id"/>
```

Comment:
This is the same identifier as inside the ratelimiting shared flow, which means both policies will using the same counter. This should be changed to something that uniquely identifies the binary path (perhaps per-application?).

Reply:
You're right, we want calls to Binary and Non-Binary endpoints to be 'mutually exclusive' in terms of rate limiting, which is what we have currently. If I use the same ref, this won't happen. I'll have to think what flow variable we can use here.

Reply:
I chose to add 
<Identifier ref="app.asid"/>
which is a custom attribute identifying the accredited system. At the moment, there is one unique per app, so I hope this will make the counters be mutually exclusive.